hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestFileSizeCountTask.java

```diff
@@ -73,45 +73,60 @@ public void setUp() {
   }
 
   @Test
-  public void testReprocess() throws IOException {
-    OmKeyInfo omKeyInfo1 = mock(OmKeyInfo.class);
-    given(omKeyInfo1.getKeyName()).willReturn("key1");
-    given(omKeyInfo1.getVolumeName()).willReturn("vol1");
-    given(omKeyInfo1.getBucketName()).willReturn("bucket1");
-    given(omKeyInfo1.getDataSize()).willReturn(1000L);
-
-    OmKeyInfo omKeyInfo2 = mock(OmKeyInfo.class);
-    given(omKeyInfo2.getKeyName()).willReturn("key2");
-    given(omKeyInfo2.getVolumeName()).willReturn("vol1");
-    given(omKeyInfo2.getBucketName()).willReturn("bucket1");
-    given(omKeyInfo2.getDataSize()).willReturn(100000L);
-
-    OmKeyInfo omKeyInfo3 = mock(OmKeyInfo.class);
-    given(omKeyInfo3.getKeyName()).willReturn("key3");
-    given(omKeyInfo3.getVolumeName()).willReturn("vol1");
-    given(omKeyInfo3.getBucketName()).willReturn("bucket1");
-    given(omKeyInfo3.getDataSize()).willReturn(1125899906842624L * 4); // 4PB
+  public void testReprocessFSO1() throws IOException {
```

Comment:
Why is this called testReprocessFSO1()?

Reply:
Thanks for the suggestion, i'll make the necessary changes!